modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,290 @@
+<2> The namespace for the `ClusterLogForwarder` CR must be `openshift-logging`.
+<3> Specify a name for the output.
+<4> Specify the `cloudwatch` type.
+<5> TBD? Optional: Specify the option CloudWatch uses to group the logs.
+<6> TBD? Optional: Specify the region or regions CloudWatch uses.
+<7> Specify the name of the secret.
+<8> Optional: Specify a name for the pipeline.
+<9> Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
+<10> Specify the name of the output to use when forwarding logs with this pipeline.
+
+. Create the CR object:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+The Red Hat OpenShift Logging Operator redeploys the Fluentd pods. If the pods do not redeploy, you can delete the Fluentd pods to force them to redeploy.
+
+[source,terminal]
+----
+$ oc delete pod --selector logging-infra=fluentd
+----
+
+'''
+
+.Example: How various configurations of the ClusterLogForwarder affect CloudWatch
+
+Suppose that you run an {product-title} cluster on AWS named `myusername-48` and the value of its `infrastructureName` is `myusername-48-7977k`.
+
+[source,terminal]
+----
+$ oc get Infrastructure/cluster -ojson | jq .status.infrastructureName
+"myusername-48-7977k"
+----
+
+The default project on the cluster is called `app`.
+
+[source,terminal]
+----
+$ oc project
+Using project "app" on server "https://api.myusername-48.devcluster.openshift.com:6443".
+----
+
+The default project has a UUID that you will reuse.
+[source,terminal]
+----
+$ oc get ns/app -ojson | jq .metadata.uid
+"794e1e1a-b9f5-4958-a190-e76a9b53d7bf"
+----
+
+The "app" project is running a Pod called `busybox` that writes a message to stdout every three seconds:
+
+[source,terminal]
+----
+$ oc run busybox --image=busybox -- sh -c 'while true; do echo "My life is my message"; sleep 3; done'
+$ oc logs -f busybox
+My life is my message
+My life is my message
+My life is my message
+...
+----
+
+In your `ClusterLogForwarder` custom resource (CR), the `all-logs` pipeline forwards `infrastructure`, `audit`, and `application` logs to the CloudWatch  output, `cw`:
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+   - name: cw
+     type: cloudwatch
+     cloudwatch:
+       region: us-east-2
+       groupBy: groupType
+     secret:
+        name: cw-secret
+  pipelines:
+    - name: all-logs
+      inputRefs:
+        - infrastructure
+        - audit
+        - application
+      outputRefs:
+        - cw
+----
+
+In CloudWatch, each AWS region contains three levels of objects:
+
+* region
+** log group
+*** log stream
+**** log event
+
+
+The three log types in the `ClusterLogForwarding` CR create three log groups in AWS:
+
+[source,terminal]
+----
+$ aws --profile osd-logging-team logs describe-log-groups | jq .logGroups[].logGroupName
```

Comment:
I should've stripped these out - please remove `--profile osd-logging-team`, that's once again something internal to Red Hat. It won't change the meaning of the rest of the command.

Reply:
Updated. Thanks.